docs/releases/overview.md

```diff
@@ -41,6 +41,54 @@ The primary focus of this release has been to add general bug fixes and improvem
+
+#### Output from nodewallet reload is now more useful JSON
+The output from `vega nodewallet reload --output=json` was not structured in a manner that was easily used. This change creates a better UX for anyone interacting with the JSON output of the command. This work was done under the issue [6549 ↗](https://github.com/vegaprotocol/vega/issues/6549).
+
+#### Rename get bundles API function `GetMultiSigSigner` to `ListMultiSigSigner`
+In order to be consistent with v2 APIs and return context aware results the get bundles API function name has been changed from `GetMultiSigSigner` to `ListMultiSigSigner`. This work was done under the issue [6458 ↗](https://github.com/vegaprotocol/vega/issues/6458).
+
+#### Swap places of PID and date in log files in the wallet service
+Before the implementation of this change wallet log files were named with the PID first e.g. `47060-2022-10-13-19-49-02.log`. This makes log files easy to search for if you have the PID but less so if you do not. In order to be able to easily sort the log files by date the file format name has been changed to start with the date e.g. `2022-10-13-19-49-02-47060.log`. This work was done under the issue [6506 ↗](https://github.com/vegaprotocol/vega/issues/6506).
+
+#### Refactor datanode API for getting balance history
+The API field `GetBalanceHistory` has been renamed to `ListBalanceHistory` and has had improvements in the documentation to help users understand APIs the 'grouping' feature. This change aslo fixes an issue with leaking account IDs. This work was done under the issue [6513 ↗](https://github.com/vegaprotocol/vega/issues/6513).
+
+#### Critical Bug fixes
+
+#### Price monitoring price-range cache restored incorrectly
+When restoring the `pricemontior` for a market from a snapshot, the integer-representation from the `wrappedDecimcal`'s used for the price-range cache are derived from the decimal representation. This is slightly different to how they are created in the normal, non-snapshot code path. This causes markets to act differently after a snapshot restore, and eventually the node falls out of consensus. This fix was implemented under issue [6525](https://github.com/vegaprotocol/vega/issues/6525).
+
+#### New features: Core
+
+#### Add reason to stopped or rejected transfer events
+In order to know why a transfer event has been stopped or rejected the transfer rejection error reason has now been exposed in `BUS_EVENT_TYPE_TRANSFER` events. This work was done under the issue [6529 ↗](https://github.com/vegaprotocol/vega/issues/6529)
+
+#### Update Tendermint to v0.34.22
```

Comment:
Will this be a breaking change for node operators, in that when they upgrade they'll also need to update tendermint?

Reply:
@jeremyletang or @ze97286 will confirm for sure but I think tendermint comes bundled as part of the Vega binary now so no action needed - They should confirm tho!

Reply:
@candida-d as far as I understand:
- from 0.54.0 (so when we next have a non-prerelease) the Validator deployment instructions will need to include the fact they **do not** need to deploy Tendermint (now built into Vega)
- Some config will need to be specified for the deployment
- Future config changes may need to be specified for future deployments